compiler/rustc_builtin_macros/src/deriving/generic/mod.rs

```diff
@@ -584,8 +584,8 @@ impl<'a> TraitDef<'a> {
                     defaultness: ast::Defaultness::Final,
                     generics: Generics::default(),
                     where_clauses: (
-                        ast::TyAliasWhereClause::default(),
-                        ast::TyAliasWhereClause::default(),
+                        ast::TyAliasWhereclause::default(),
```

Comment:
A global "Clause" -> "clause" rename is going to give a ton of mistakes like this. Can you fix those?

Reply:
Yeah I did not notice that, will be fixing it :)